frontend/src/views/camp/Invitation.vue

```diff
@@ -1,66 +1,94 @@
 <template>
-  <auth-container>
-    <div v-if="!ready" class="text-center">
-      <v-progress-circular size="64" indeterminate color="primary" />
-    </div>
-    <div v-else-if="invitationFound === true">
-      <h1 class="display-1">
-        {{ $tc('views.camp.invitation.title', 0, { campName: invitation().campTitle }) }}
+  <div class="bg">
```

Comment:
The design is now very much baked into the Invitation page. Don't we want to abstract this in a separate component (similar as authContainer). I guess you'd want to use the same layout for other pages as well (minimum for InvitationRejected and InvitationUpdateError).

Is the idea to use the same layout for the login page as well?

Reply:
I think I would love a different design for different concepts. E.g. I have already an idea for the 404 page.